### PR TITLE
Add a warning about using sudo for a TCP sample

### LINKED ISSUE #3579: tcp-echo example should run docker with sudo
Our test command at https://preliminary.istio.io/docs/tasks/traffic-management/tcp-traffic-shifting/ is this:

for i in {1..10}; do \
docker run -e INGRESS_HOST=$INGRESS_HOST -e INGRESS_PORT=$INGRESS_PORT -it --rm busybox sh -c "(date; sleep 1) | nc $INGRESS_HOST $INGRESS_PORT"; \
done

But for docker to work (on this machine at least) I have to run it as "sudo docker".    That makes the example work.

However, for what it's worth, this test command seems to work equally well:

$ for i in {1..10}; do sh -c "(date; sleep 1) | busybox nc  $INGRESS_HOST $INGRESS_PORT"; done
one Fri Mar  8 18:12:05 GMT 2019
one Fri Mar  8 18:12:06 GMT 2019
one Fri Mar  8 18:12:07 GMT 2019
one Fri Mar  8 18:12:08 GMT 2019
one Fri Mar  8 18:12:09 GMT 2019
one Fri Mar  8 18:12:10 GMT 2019
one Fri Mar  8 18:12:11 GMT 2019
one Fri Mar  8 18:12:12 GMT 2019
one Fri Mar  8 18:12:13 GMT 2019
one Fri Mar  8 18:12:14 GMT 2019

If you want to use the "real" nc instead of Busybox you have to use the -N flag to avoid an indefinite hang:

$ for i in {1..10}; do sh -c "(date; sleep 1) | nc -N  $INGRESS_HOST $INGRESS_PORT"; done
one Fri Mar  8 18:12:59 GMT 2019
one Fri Mar  8 18:13:00 GMT 2019
one Fri Mar  8 18:13:01 GMT 2019
one Fri Mar  8 18:13:02 GMT 2019
one Fri Mar  8 18:13:03 GMT 2019
one Fri Mar  8 18:13:04 GMT 2019
one Fri Mar  8 18:13:05 GMT 2019
one Fri Mar  8 18:13:06 GMT 2019
one Fri Mar  8 18:13:07 GMT 2019
one Fri Mar  8 18:13:08 GMT 2019

### PR DESCRIPTION
**Pull request by venilnoronha**
This adds a warning about using `sudo` while running the TCP Traffic Shifting sample on certain platforms. Fixes #3579.

### CODE REVIEW

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 1]
```diff
@@ -86,6 +86,10 @@ the [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port) doc
     one Mon Nov 12 23:25:19 UTC 2018
     {{< /text >}}
 
+    {{< tip >}}
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]:
```suggestion
    {{< warning >}}
```

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 2]
```diff
@@ -86,6 +86,10 @@ the [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port) doc
...
+    You may need to prefix the `docker` command with `sudo` depending on your Docker installation.
+    {{< /tip >}}
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]:
```suggestion
    {{< /warning >}}
```

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 3]
```diff
@@ -86,6 +86,10 @@ the [Bookinfo](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port) doc
     one Mon Nov 12 23:25:19 UTC 2018
     {{< /text >}}
 
+    {{< tip >}}
+    You may need to prefix the `docker` command with `sudo` depending on your Docker installation.
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]:
```suggestion
    The `docker` command may require using `sudo` depending on your Docker installation.
```

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 4]
```diff
@@ -144,6 +148,10 @@ was routed to the `v1` version of the `tcp-echo` service.
     one Mon Nov 12 23:39:07 UTC 2018
     {{< /text >}}
 
+    {{< tip >}}
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]:
```suggestion
    {{< warning >}}
```

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 5]
```diff
@@ -144,6 +148,10 @@ was routed to the `v1` version of the `tcp-echo` service.
...
+    You may need to prefix the `docker` command with `sudo` depending on your Docker installation.
+    {{< /tip >}}
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]:
```suggestion
    {{< /warning >}}
```

File: content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md [hunk 6]
```diff
@@ -144,6 +148,10 @@ was routed to the `v1` version of the `tcp-echo` service.
     one Mon Nov 12 23:39:07 UTC 2018
     {{< /text >}}
 
+    {{< tip >}}
+    You may need to prefix the `docker` command with `sudo` depending on your Docker installation.
```
Comment (rcaballeromx) [inline content/docs/tasks/traffic-management/tcp-traffic-shifting/index.md]:
```suggestion
    The `docker` command may require using `sudo` depending on your Docker installation.
```

**Comment by venilnoronha** [comment]
@rcaballeromx I've addressed your comments in c6fc37b.

**Pull request merged**